scripts/001-binutils.sh

```diff
@@ -27,8 +27,13 @@ rm -rf build-$TARGET && mkdir build-$TARGET && cd build-$TARGET || { exit 1; }
   --quiet \
   --prefix="$PS2DEV/$TARGET_ALIAS" \
   --target="$TARGET" \
+  --enable-interwork \
+  --enable-plugins \
   --disable-lto \
+  --disable-nls \
+  --disable-separate-code \
```

Comment:
This will change the generated code, but is a good thing to do I think.

Reply:
Me too